eth/stagedsync/stage_snapshots.go

```diff
@@ -466,7 +466,6 @@ func FillDBFromSnapshots(logPrefix string, ctx context.Context, tx kv.RwTx, dirs
 				return err
 			}
 			_ = tx.ClearBucket(kv.MaxTxNum)
-			hasInsertedAtLeastOneTxNum := true
```

Comment:
it is really strange variable, it's always true.. But maybe there are no other way to do this?

Reply:
maybe it was supposed to be initialised with false?

Reply:
does it makes sense if ForceWrite is replaced by Append?

Reply:
idk, it is literally iterating one block by one (over their bodies). so, there should be append instead. 
Sad to know, that we don't have any test for this and now I am testing append version on testnets(Amoy has checked)